options/internal/generic/global-config.cpp

```diff
@@ -0,0 +1,30 @@
+#include <stdlib.h>
+#include <string.h>
+#include <mlibc/global-config.hpp>
+
+namespace mlibc {
+
+struct GlobalConfigGuard {
+	GlobalConfigGuard();
+};
+
+GlobalConfigGuard guard;
+
+GlobalConfigGuard::GlobalConfigGuard() {
+	// Force the config to be created during initialization of libc.so.
+	mlibc::globalConfig();
+}
+
+static bool envEnabled(const char *env) {
+	auto value = getenv(env);
+	if (value)
+		return !strcmp(value, "1");
```

Comment:
It might be nicer to check for any nonzero, nonnull value (so, `return value && *value;`)